site/layouts/shortcodes/card.html

```diff
@@ -22,9 +22,11 @@
 {{- $rtl := .Get "rtl" | default false -}}
 
 <div class="card{{ if eq $borders false }} border-0{{ end }}{{ if and $borders (eq $background "dark") }} border-dark{{ end }}">
-  <svg class="bd-placeholder-img card-img-top" width="100%" height="162" xmlns="http://www.w3.org/2000/svg" role="img" aria-label="{{ if $rtl }}تعليق على الصورة{{ else }}Image cap{{ end }}" preserveAspectRatio="xMidYMid slice" focusable="false">
+  <svg class="bd-placeholder-img card-img-top" width="100%" height="169" xmlns="http://www.w3.org/2000/svg" role="img" aria-label="{{ if $rtl }}العنصر النائب: شرح الصورة{{ else }}Placeholder: Image caption{{ end }}" preserveAspectRatio="xMidYMid slice" focusable="false">
     <rect width="100%" height="100%" fill="{{ (index $.Site.Data.grays 3).hex }}"/>
-    <text x="50%" y="50%" fill="{{ (index $.Site.Data.grays 5).hex }}" dy=".3em">{{ if $rtl }}تعليق على الصورة{{ else }}Image cap{{ end }}</text>
+    <svg x="30%" y="30%" width="40%" height="40%" viewBox="0 0 24 24" fill="#00000014" aria-hidden="true" focusable="false">
```

Comment:
We don't have a corresponding value in our `$.Site.Data.grays` list?

Reply:
No, I don't think so. I chose to have a kind of watermark with opacity since we have some use cases using colored backgrounds and the grey was pretty ugly to use with. Tell me what you think the best between:
![image](https://user-images.githubusercontent.com/91960143/225570293-d2794b16-341e-4352-b0ad-18ea51b43b19.png)
![image](https://user-images.githubusercontent.com/91960143/225570416-bf2f9c58-7d46-459d-ac1b-fdc0bcade2a8.png)

Reply:
None 😄 Should probably be a dark gray but nevermind, it's not that related to the PR. Edge case.